test/unit/runner.test.js

```diff
@@ -128,16 +128,21 @@ describe('runner', function () {
     describe('extractRunnableItems', function () {
```

Comment:
Leave existing tests alone, or move them to their own `describe` block for folders.

Reply:
Moved `entrypoint` as folder and request into individual `describe` blocks.